packages/scipy/patches/force_malloc.patch

```diff
@@ -0,0 +1,94 @@
+commit 7a5b003922134abd01ae33b0a3e84d006673e9c6
+Author: Roman Yurchak <rth.yurchak@pm.me>
+Date:   Mon Oct 8 12:13:55 2018 +0200
+
+    Workaround for malloc issues
+    
+    This aims to avoid the "invalid call target: $_malloc" issues
+    
+      - fixes scipy/optimize/zeros.c
+      - skips scipy/integrate/dop.pyf
+      - skips scipy/optimize/cobyla
```

Comment:
Fixing the malloc issue in `zeros.c` was simple enough. In the other two it fails in f2c code, so malloc would need to be done in Fortran I couldn't find an obvious place where to put it, but I have not investigated this that much.